config/runtime.exs

```diff
@@ -162,3 +137,30 @@ config :logger, :error_log, path: "#{log_root_path}/error.log"
 config :logger, :notice_log, path: "#{log_root_path}/notice.log"
 
 config :logger, :info_log, path: "#{log_root_path}/info.log"
+
+enable_discord_bridge =
+  Teiserver.ConfigHelpers.get_env("TEI_ENABLE_DISCORD_BRIDGE", false, :bool)
```

Comment:
I changed the default from `true` to `false` so that it works out of the box locally. This should not impact anything on the real servers because [the env var](https://github.com/beyond-all-reason/ansible-teiserver/blob/b42a42da2726ba0ac9f64d3850ceb7df972a1657/roles/teiserver/templates/teiserver.env.j2#L40-L48) is always set correctly.